source/config.js

```diff
@@ -55,6 +55,9 @@ const defaultConfig = {
   // Instead of launching ungit with the current folder force a different path to be used. Can be set to null to force the home screen.
   forcedLaunchPath: undefined,
 
+  // Lock the path and consequently turn off the repository browser.
```

Comment:
Can we add a description about how this field is related to the `forcedLaunchPath`?